web/play/src/Tailwind/Breakpoints.elm

```diff
@@ -0,0 +1,29 @@
+module Tailwind.Breakpoints exposing (..)
```

Comment:
Why this file and the other one in the Tailwind directory are commited ? They seems generated.

Reply:
i remove all generated files and add a command to recreate it